index.html

```diff
@@ -78,6 +78,102 @@ <h3>HTML 4.0 (1997-1999)</h3>
         </ul>
       </section>
       <hr />
+      <section>
+        <h2>The Rise of Modern HTML</h2>
+        <h3>XHTML (2000)</h3>
+        <p>
+          A stricter version of HTML based on <strong>XML</strong>, designed for
+          cleaner coding. However, it was considered too rigid and never gained
+          wide adoption.
+        </p>
+        <h3>HTML5 (2014 - Present)</h3>
+        <p>The most significant update to HTML, bringing:</p>
+        <p><strong>Key Features of HTML5:</strong></p>
+        <ol>
+          <li><strong> Native multimedia</strong> support</li>
+          <li><strong> Semantic elements</strong></li>
+          <li><strong> Canvas and SVG</strong> for graphics and animations</li>
+          <li><strong> Improved forms</strong></li>
+          <li>
+            <strong> Offline support</strong> through
+            <strong>Service Workers</strong>
+          </li>
+        </ol>
+        <hr />
+        <p>
+          As HTML evolved, the way web pages were processed and displayed became
+          more sophisticated. This led to the development of the
+          <strong>Document Object Model (DOM)</strong> and advanced
+          <strong>Rendering Engines</strong>, which play a crucial role in how
+          HTML, CSS, and JavaScript work together.
+        </p>
+        <h3>Document Object Model (DOM)</h3>
+        <p>
+          The <strong>DOM</strong> represents an HTML document as a tree-like
+          structure where every element (tags, attributes, and text) is treated
+          as an object. This allows JavaScript to dynamically interact with the
+          page by modifying elements in real time.
+        </p>
+        <p>For example, a JavaScript script can:</p>
+        <ul>
+          <li>
+            Change the text inside an
+            <font color="green"><code>&lt;h1&gt;</code></font> tag
+          </li>
+          <li>Modify CSS properties dynamically</li>
+          <li>Add or remove elements from the page</li>
+        </ul>
+        <h3>Rendering Engines</h3>
+        <p>
+          Each web browser has its own <strong>rendering engine</strong> that
+          interprets HTML, CSS, and JavaScript to display a web page. Some of
+          the most well-known engines include:
+        </p>
+        <ul>
+          <li><strong>Blink</strong> (used by Chrome and Edge)</li>
+          <li><strong>WebKit</strong> (used by Safari)</li>
+          <li><strong>Gecko</strong> (used by Firefox)</li>
+        </ul>
+        <p>These engines follow a structured process:</p>
+        <ol>
+          <li><strong> Parsing HTML and CSS</strong> into a render tree</li>
+          <li>
+            <strong> Layout calculation</strong> to determine element positions
+          </li>
+          <li><strong> Painting</strong> the final output on the screen</li>
```

Comment:
Some extra spaces here, too:
```suggestion
          <li><strong>Parsing HTML and CSS</strong> into a render tree</li>
          <li>
            <strong>Layout calculation</strong> to determine element positions
          </li>
          <li><strong>Painting</strong> the final output on the screen</li>
```

Reply:
Same thing here with this ordered list - wanted it to more closely resemble the Google Doc.